src/test/scala/chiselTests/aop/InjectionSpec.scala

```diff
@@ -132,10 +131,10 @@ class InjectionSpec extends ChiselFlatSpec with Utils {
   "Test" should "fail if inserted the wrong values" in {
     assertTesterFails { new AspectTester(Seq(9, 9, 9)) }
   }
-  "Test" should "pass if pass wrong values, but correct with aspect" in {
+  "Test" should "pass if pass wrong values, but correct with aspect" ignore {
```

Comment:
Can we put a TODO here to say Injecting aspect needs to be handled purely in chisel?